src/platform_impl/linux/event_loop.rs

```diff
@@ -574,6 +576,25 @@ impl<T: 'static> EventLoop<T> {
               Inhibit(false)
             });
 
+            let tx_clone = event_tx.clone();
+            window.connect_scroll_event(move |_, event| {
+              let (x, y) = event.delta();
+              if !matches!(event.direction(), ScrollDirection::Smooth) {
+                if let Err(e) = tx_clone.send(Event::WindowEvent {
+                  window_id: RootWindowId(id),
+                  event: WindowEvent::MouseWheel {
+                    device_id: DEVICE_ID,
+                    delta: MouseScrollDelta::LineDelta(x as f32, y as f32),
+                    phase: TouchPhase::Moved,
```

Comment:
I believe there are more variants can be added. But I don't mind there's only one at first.
Just add a TODO comment to remind us future work.

Reply:
Hi,

Only `ScrollDirection::Smooth` and `ScrollDirection::{Up, Right, Down, Left}` are emitted on my machine, so I mapped `ScrollDirection::Smooth` to `TouchPhase::Moved` and other `ScrollDirection` to `TouchPhase::Ended` on commit f1f6f35.

How to test:

```
cargo run --example window
```

Expected result:

```
WindowEvent { window_id: WindowId(WindowId(1)), event: MouseWheel { device_id: DeviceId(DeviceId(0)), delta: LineDelta(0.0, 1.0), phase: Moved, modifiers: (empty) } }
WindowEvent { window_id: WindowId(WindowId(1)), event: MouseWheel { device_id: DeviceId(DeviceId(0)), delta: LineDelta(0.0, 0.0), phase: Ended, modifiers: (empty) } }
```